### ISSUE TITLE
Any license?

### ISSUE THREAD

**Issue by yrashk** [Strytyp/wasm-nom#1]
Hi,

Thank you for putting this project together. Is this code licensed under any license? I am considering using it, and I might also need to make further changes to it (either contributed or as a fork), specifically `no_std` support being of the utmost importance (cursory look suggests that this shouldn't be too hard).

**Comment by fabrizio-m** [comment]
Hi, I just added a MIT license, I think it should be enough but tell me if somethings is missing.
I do no have plans for this project right now so I have no problem with contributions.

**Comment by fabrizio-m** [comment]
Indeed, it looks like it is easy to add support for no_std, I have created an example branch using it.
